validator/testcases/javascript/instanceproperties.py

```diff
@@ -225,6 +252,8 @@ def set_contentScript(value, traverser):
                         "is dangerous and error-prone. Please use a separate "
                         "JavaScript file, along with the "
                         "`contentScriptFile` property instead.",
+            signing_help="Please do not use the `contentScript` property "
+                         "in any add-ons submitted for automated signing.",
```

Comment:
Side-loaded add-ons are reviewed in the same way as listed add-ons. Other unlisted add-ons are only reviewed if they don't pass automated validation, and then, only to verify that the warnings are safe (unless they happen to look suspicious and get a heavier review). But, since `contentScript` properties can't be verified as safe without manual review, and also aren't at all necessary, they aren't allowed for unlisted add-ons.